soql-environment/src/main/scala/com/socrata/soql/environment/TableName.scala

```diff
@@ -4,27 +4,25 @@ case class TableName(name: String, alias: Option[String] = None) {
   import TableName._
 
   override def toString(): String = {
-    alias.map(removeSodaPrefix).foldLeft(replaceSodaPrefix(name))((n, a) => s"$n AS $a")
+    alias.map(removePrefix).foldLeft(replacePrefixWithAt(name))((n, a) => s"$n AS $a")
   }
 
   def qualifier: String = alias.getOrElse(name)
 }
 
 object TableName {
-  val PrimaryTable = TableName("_")
-
-  val Prefix = "@"
-
-  val Field = "."
-
   // All resource name in soda fountain have an underscore prefix.
   // that is not exposed to end users making SoQLs.
   // To handle this mis-match, we automatically prepend the prefix when the parse tree is built.
   // To remove this "feature", re-define this with an empty string "" or completely remove this variable.
-  val SodaFountainTableNamePrefix = "_"
-  val SodaFountainTableNamePrefixSubStringIndex = 1
+  val SodaFountainPrefix = "_"
+  val PrimaryTable = TableName(SodaFountainPrefix)
+  val Prefix = "@"
+  val Field = "."
+  val PrefixIndex = 1
 
-  def withSodaFountainPrefix(s: String) = s"$SodaFountainTableNamePrefix$s"
-  def replaceSodaPrefix(s: String) = s.replaceFirst(TableName.SodaFountainTableNamePrefix, TableName.Prefix)
-  def removeSodaPrefix(s: String) = s.replaceFirst(TableName.SodaFountainTableNamePrefix, "")
+  def removePrefix(s: String): String = s.substring(PrefixIndex)
+  def addAtPrefix(s: String): String = s"$Prefix$s"
```

Comment:
This function and the next have very different names with similar functionality. Maybe name this one `withAtPrefix`?